src/app/features/date-range/services/api/api.service.ts

```diff
@@ -0,0 +1,13 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { UnavailablePeriods } from '../../models';
+
+@Injectable()
+export class ApiService {
```

Comment:
honestly to say i think we do not need this service here.

Reply:
done